ember-file-upload/src/test-support.ts

```diff
@@ -91,6 +91,60 @@ export async function dragAndDrop(
   return triggerEvent(dropzone, 'drop', { dataTransfer });
 }
 
+interface FileSystemEntryStub {
+  isFile: boolean;
+  file: (callback: (file: File | Blob) => void) => void;
+}
+
+export async function dragAndDropDirectory(
+  selector: string,
```

Comment:
Let's follow the same convention as the other test helpers and allow this to be a selector or an element.

```suggestion
    target: string | HTMLElement,
```

Reply:
Updated